lib/libutils/isoc/include/stdio.h

```diff
@@ -32,6 +32,15 @@
 
 typedef struct _FILE FILE;
 
+#if defined(__KERNEL__)
+#include <printk.h>
+
+#define snprintf(_fmt, ...) snprintk((_fmt), __VA_ARGS__)
```

Comment:
Ah, my bad. I suppose I can drop this and just use the `printk` functions directly in `trace.c`. Will push an update shortly.